helmfile.d/stacks/tekton.yaml

```diff
@@ -0,0 +1,17 @@
+---
+templates:
+  tekton-pipelines:
+    disableValidationOnInstall: true
+    condition: ck8sManagementCluster.enabled
+    installed: {{ .Values | get "tektonPipelines.enabled" false }}
+    namespace: tekton-pipelines
+    name: tekton-pipelines
+    labels:
+      app: tekton
+    chart: charts/tekton-pipelines
+    version: 0.1.0
+    needs:
+      - kube-system/admin-namespaces
+      - monitoring/kube-prometheus-stack # creates servicemonitors and prometheusrules
```

Comment:
Add psps as a needs.
(While it technically doesn't need it I feel like it will be good to have the namespace fully prepared before installing pipelines.)